src/Pages/Form/Components/PatientInformationSection.tsx

```diff
@@ -0,0 +1,265 @@
+              </Box>
+            </ToggleButton>
+            <ToggleButton value="female" className={classes.button}>
+              <Box display="flex">
+                <img src={FemaleIcon} alt="female icon" />
+              </Box>
+            </ToggleButton>
+          </GenderStyledToggleButtonGroup>
+        </Grid>
+        <Box display="flex">
+          <Box className={classes.boxMale}>
+            <Typography variant="body1">Male</Typography>
+          </Box>
+          <Box className={classes.boxFemale}>
+            <Typography variant="body1">Female</Typography>
+          </Box>
+        </Box>
+        {/* Arrival Time - Date & Time Picker*/}
+        <Box className={classes.textTitle}>
+          <Typography variant="h4">Arrival Time</Typography>
+        </Box>
+        <Grid container spacing={7}>
+          <MuiPickersUtilsProvider utils={DateFnsUtils}>
+            <Grid item>
+              <KeyboardDatePicker
+                variant="inline"
+                format="dd/MM/yyyy"
+                label="Arrival Date"
+                value={selectedDate}
+                onChange={handleDateChange}
+                KeyboardButtonProps={{
+                  "aria-label": "change date",
+                }}
+                className={classes.picker}
+              />
+            </Grid>
+            <Grid item>
+              <KeyboardTimePicker
+                variant="inline"
+                label="Arrival Time"
+                value={selectedDate}
+                onChange={handleDateChange}
+                KeyboardButtonProps={{
+                  "aria-label": "change time",
+                }}
+                className={classes.picker}
+              />
+            </Grid>
+          </MuiPickersUtilsProvider>
+        </Grid>
+        {/* Onset - Button */}
+        <Box className={classes.textTitle}>
+          <Typography variant="h4">Onset</Typography>
+        </Box>
+        <Grid container spacing={3}>
+          <OnsetStyledToggleButtonGroup
+            value={onset}
+            exclusive
+            onChange={handleOnset}
+          >
+            <ToggleButton value="clearOnset" className={classes.button}>
+              <Box display="flex">
+                <Typography variant="subtitle1">Clear onset</Typography>
+              </Box>
+            </ToggleButton>
+            <ToggleButton value="unknownOnset" className={classes.button}>
+              <Box display="flex">
+                <Typography variant="subtitle1">Unknown onset</Typography>
+              </Box>
+            </ToggleButton>
+          </OnsetStyledToggleButtonGroup>
+        </Grid>
+        {/* Upload CT Scan - Button*/}
+        <Box className={classes.textTitle}>
+          <Typography variant="h4">Upload CT Scan</Typography>
+        </Box>
+        <Button variant="contained" className={classes.buttonUpload}>
+          <Typography variant="subtitle1">Upload</Typography>
```

Comment:
For upload component, you can use `TextField type="file"`.

Reply:
Text field was hard to style. I use Button with input instead and it works the same.